03-seeking-help.Rmd

```diff
@@ -124,6 +124,11 @@ your issue.
 > What is the difference between the `sep` and `collapse` arguments?
 > 
 
+> ## Challenge 3 {.challenge}
+> Use help to find out which function and with what parameters should you use
```

Comment:
Are you sure about that? These are the parameters that you need to use if you want the import to work right. I'm not native English speaker though.